components/spinner.tsx

```
@@ -0,0 +1,31 @@
+import { Loader } from "lucide-react";
+
+import { cva, type Variantprops } from "class-variance-authority";
```

ESLint

[error] 'Variantprops' is defined but never used.

(@typescript-eslint/no-unused-vars)